src/tribol/interface/mfem_tribol.cpp

```diff
@@ -317,7 +331,7 @@ void updateMfemParallelDecomposition()
       // creates a new redecomp mesh based on updated coordinates and updates
       // transfer operators and displacement, velocity, and response grid
       // functions based on new redecomp mesh
-      mfem_data->UpdateMfemMeshData();
+      mfem_data->UpdateMfemMeshData( coupling_scheme.getParameters().binning_proximity_scale );
```

Comment:
Why does the binning proximity scale need to be sent here. Shouldn't the binning proximity live on the coupling scheme and the coupling scheme drives mesh interactions? What is UpdateMfemMeshData() doing that needs the binning proximity scale?

Reply:
The redecomp mesh needs the binning proximity.  It's created in this method.